docs/ru/assemble_4.md

```diff
@@ -0,0 +1,336 @@
+2. Стрелки на полетном контроллере и центральной деке должны быть направлены в одну сторону.
+3. Подключите шлейф питания PDB к разъему *"POWER"* полетного контроллера, закрутив его в "косичку" для взаимной фиксации проводов.
+
+    <center>
+        <img src="../assets/4/18.png" width=300 class="zoom border">
+        <img src="../assets/4/18_1.png" width=300 class="zoom border">
+    </center>
+
+4. Установите 4 алюминиевые стойки 40 мм с помощью винтов М3х10.
+
+    <center>
+        <img src="../assets/4/19.png" width=300 class="zoom border">
+        <img src="../assets/4/19_1.png" width=300 class="zoom border">
+    </center>
+
+5. Подключите сигнальные провода регуляторов к полетному контроллеру следующим образом:
+
+    <center>
+        <img src="../assets/4/20.png" width=300 class="zoom border">
+        <img src="../assets/4/20_1.png" width=300 class="zoom border">
+    </center>
+
+6. Установите 2 стойки "мама-мама" 15 мм на центральную деку с помощью винтов М3х8.
+7. Другие 2 стойки крепятся винтами М3х10 при том, что фиксируют 2 карбоновые детали, помимо стойки.
+
+    <center>
+        <img src="../assets/4/21.png" width=300 class="zoom border">
+        <img src="../assets/4/21_1.png" width=300 class="zoom border">
+    </center>
+
+## Установка обруча для светодиодной ленты
+
+1. Согните поликарбонатную заготовку в обруч и зафиксировать ее концы в замке.
+2. Установите обруч на раму, используя пазы.
+
+    <center>
+        <img src="../assets/4/22.png" width=300 class="zoom border">
+        <img src="../assets/4/23.png" width=300 class="zoom border">
+    </center>
+
+## Установка Raspberry Pi
+
+1. Установите плату Raspberry Pi на стойки, используя 4 стойки "папа-мама".
+2. Протяните провода от BEC через паз в центральной раме.
+
+    <center>
+        <img src="../assets/4/24.png" width=300 class="zoom border">
+        <img src="../assets/4/26.png" width=300 class="zoom border">
+    </center>
+
+3. Подключите провод питания от BEC к Raspberry, согласно схеме:
+
+<img src="../assets/4/26_1.png" width=300 class="zoom center border">
+
+## Установка светодиодной ленты на обруч
+
+1. Проверьте наличие напаянных пинов на контактах ленты (при отсутствии - напаять).
+2. Установите светодиодную ленту на обруч (используя клеевой слой на ленте) так, чтобы контакты были в задней части коптера. Для фиксации использовать **стяжки**.
+
+    <center>
+        <img src="../assets/4/27.png" width=300 class="zoom border">
+        <img src="../assets/4/27_1.png" width=300 class="zoom border">
+        <img src="../assets/4/28.png" width=300 class="zoom border">
+    </center>
+
+## Подключение светодиодной ленты к Raspberry Pi
+
+1. Питание для ленты берется от второго BEC. Подключите контакты *«-»* и *«+»* к *Ground* и *5v* на ленте соответственно.
+2. Контакт *D* можно подключить к любому свободному пину "GPIO" на Raspberry.
+
+<img src="../assets/4/31_1.png" width=300 class="zoom center border">
```

Comment:
This image (and other images that follow) shows the LED strip connected to GPIO18, while we tell the users to connect it to GPIO21. This is confusing and should be fixed.